job_bundles/README.md

```diff
@@ -0,0 +1,308 @@
+and an intuitive graphical Job submission interface. For example, after 
+[configuring the Deadline Cloud CLI](https://github.com/aws-deadline/deadline-cloud/blob/mainline/README.md#configuration) 
+you can run it with the Blender sample in this samples directory (`deadline bundle gui-submit blender_render/`) to see:
+
+![UI Shared Settings](../.images/blender_submit_shared_settings.png) ![UI Job Settings](../.images/blender_submit_job_settings.png) ![UI Job Attachments](../.images/blender_submit_job_attachments.png) 
+
+where the contents of the Job-specific settings panel are automatically generated using the `userInterface` properties of Job Parameters
+[defined in the Job Template](https://github.com/aws-deadline/deadline-cloud-samples/blob/bdd5ff5ea29eb7457c9a78ba39166b891b79151e/job_bundles/blender_render/template.yaml#L11-L19) within 
+your Job Bundle, and the [Queue Environments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue-environment.html)
+defined on the Queue that you are submitting to.
+
+If you prefer a non-interactive submission workflow for your Job Bundles, then you can submit this Job Bundle with the command
+`deadline bundle submit --yes --name Demo -p BlenderSceneFile=<location-of-your-scene-file> -p OutputDir=<file-path-for-job-outputs> blender_render/`
+or use the `deadline.client.api.create_job_from_job_bundle` function in the [`deadline` Python package](https://github.com/aws-deadline/deadline-cloud).
+
+All of the Job submitter plugins that have been developed by the AWS Deadline Cloud team, such as the 
+[Autodesk Maya plugin](https://github.com/aws-deadline/deadline-cloud-for-maya), are simply generating a Job Bundle for your
+Job submission and then using the [Deadline Cloud Python package](https://github.com/aws-deadline/deadline-cloud) to
+submit your Job to Deadline Cloud. You can see the Job Bundles that are
+submitted by looking in the job history directory on your workstation after submitting a Job using a submitter plugin or the Deadline
+Cloud CLI. You can find your job history directory by running the command: `deadline config get settings.job_history_dir`.
+
+## Elements of a Job Bundle
+
+A Job Bundle is a directory structure that contains at least an
+[Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) file, but may contain
+other files as follows:
+
+```
+/template.yaml (or template.json)
+/asset_references.yaml (or asset_references.json)
+/parameter_values.yaml (or parameter_values.json)
+<plus any other Job-specific files that you'd like>
+```
+
+The only required file is the Job Template (`template.yaml`/`template.json`) file that describes the structure and behaviour
+of your Job. The other files are optional, and are described in the following subsections.
+
+### Elements - Job Template
+
+The `template.yaml`/`template.json` file defines the runtime environment and all the processes that will run
```

Comment:
this whole doc is a great start. It's not clear how deep we want to go but we're missing mentioning some key features like steps(barely mentioned), step dependencies, and job environments.

Reply:
Yes. I'm trying to keep this one narrowly focused on the delta between a Job Template and a Job Bundle. Otherwise, it can get quite out of hand if I try to explain everything -- I'd have to include a lot of details about path mapping, storage profiles, and job attachments when talking about asset references later. 

The plan is a series of documents that explain the various aspects of the system. One for Job Templates, one for Job Bundles (this one), and one for "assets" (path mapping, storage profiles, and job attachments).